runtime/autoload/dist/ft.vim

```diff
@@ -557,17 +557,38 @@ export def FTm()
 enddef
 
 export def FTmake()
-  # Check if it is a Microsoft Makefile
+  # Check if it is a BSD, GNU, or Microsoft Makefile
+  # - 2: detected according to file name
+  # - 1: detected according to file content
```

Comment:
Huh. I didn't know one could use `"` instead of `#` for comments, but it's everywhere in that file, so fine for me.

Reply:
That is a vim9script file and thus a comment starts with `#`.